easybuild/framework/easyblock.py

```diff
@@ -1765,7 +1765,12 @@ def install_extensions(self, install=True):
 
         if build_option('parallel_extensions_install'):
             self.log.experimental("installing extensions in parallel")
-            self.install_extensions_parallel(install=install)
+            try:
+                self.install_extensions_parallel(install=install)
+            except NotImplementedError:
+                # If parallel extension install is not supported for this type of extension then install sequentially
+                self.log.experimental("parallel extensions install is not supported for ..., so install sequentially.")
```

Comment:
I assume this is WIP, and the `...` still needs to be replaced with something more meaningful?

Reply:
Yes, though I cannot remember what I was intending to put there. So I've changed it in 3e5115a